components/Form/FormSelect.js

```diff
@@ -10,6 +10,30 @@ import { isDevEnv } from "~/config/utils/env";
 
 const animatedComponents = makeAnimated();
 
+// This is a seldom used control. It should only be used
+// in cases where you do not want click events on your Select Control.
+// e.g. Showing a modal onClick instead of the default dropdown.
+// You will need to define your click event in a wrapper component.
+export const CustomSelectControlWithoutClickEvents = ({
```

Comment:
This is not "pretty" but the path of least resistance to override touch and click events on react-select.
It is only used for legacy reasons because some pages have a particular look and feel and are using react-select.